### PR TITLE
Merge rc sim joy

### PR DESCRIPTION
**Pull request by bsutherland333**
This pull request simplifies using an rc transmitter with the simulation by merging the rc_joy and rc_sim utilities and replacing the enable/disable service calls with toggle service calls. It also removed the command_joy.py utility as this duplicates a lot of code and it would be better to integrate this into rc.py, if that functionality is ever needed again.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,35 +43,23 @@ fixedwing simulation and `ros2 launch rosflight_sim multirotor.launch.py` for a
...
-
-### Simulator, rosflight_io, and rc_joy launch files
+### rc
+
+This script contains a node that allows for connecting a gamepad or transmitter to the simulator or using a simulated controller if neither of those are available.
+
+To use a transmitter or controller, plug in a controller and launch `ros2 run rosflight_sim rc.py --ros-args --remap
+RC:=/fixedwing/RC` for a fixedwing sim or `ros2 run rosflight_sim rc.py --ros-args --remap RC:=/multirotor/RC` for a
+multirotor sim. Currently supported devices are Taranis Q-X7 transmitters, XBox controllers, RealFlight InterLink
+controllers, and RadioMaster TX16S transmitters. Adding addition devices can be done easily, so long as the device has
```
Comment (iandareid) [inline README.md:53]: Minor typo in "addition".

File: README.md [hunk 2]
```diff
@@ -43,35 +43,23 @@ fixedwing simulation and `ros2 launch rosflight_sim multirotor.launch.py` for a
...
+USB gamepad support.
```
Comment (iandareid) [inline README.md:54]: Should we point them to the documentation for adding a controller?

Comment (bsutherland333) [reply README.md:54]: Haha well there is no documentation... It's a easy thing to do though, I'll add a quick explanation of how to do so here.

Comment (iandareid) [reply README.md:54]: Oh no, I'll put that on the  list for docs.

**Comment by bsutherland333** [comment]
For now, I just added instructions to follow what already is being done to add additional devices in the rc_sim.py script since it is really easy to do. If we add a webpage for this later then we can update the readme to reference it.

**Review by JMoore5353 — APPROVED** [review]

**Pull request merged**